packages/ppbgdi/data_stream/cloudfront_logs/elasticsearch/ingest_pipeline/default.yml

```diff
@@ -676,4 +699,7 @@ on_failure:
       value: pipeline_error
   - append:
       field: error.message
-      value: '{{{ _ingest.on_failure_message }}}'
+      value: >-
+        Processor '{{ _ingest.on_failure_processor_type }}'
+        {{#_ingest.on_failure_processor_tag}}with tag '{{ _ingest.on_failure_processor_tag }}'
+        {{/_ingest.on_failure_processor_tag}}failed with message '{{ _ingest.on_failure_message }}'
```

Comment:
not sure what `{{#_ingest.on_failure_processor_tag}}` and `{{/_ingest.on_failure_processor_tag}}` does but I assume it's ok.

Reply:
This is a new pattern introduced by elastic that will be rolled out to all error handlers. This helps to identify which one of the processors of the same type failed. Adding processor tags are only used in the context of the ingest pipeline itself, e.g. error handlers and metrics and are optional.